Sources/MySQLKit/MySQLDialect.swift

```diff
@@ -43,7 +43,7 @@ public struct MySQLDialect: SQLDialect {
     public func customDataType(for dataType: SQLDataType) -> SQLExpression? {
         switch dataType {
         case .text:
-            return SQLRaw("VARCHAR(255)")
+            return SQLRaw("TEXT")
```

Comment:
The default here is `TEXT`: https://github.com/vapor/sql-kit/blob/master/Sources/SQLKit/Query/SQLDataType.swift#L29

So we can just remove this switch entirely and always return `nil`.

Reply:
Won't the benchmark test still fail because it is creating a field with a default value?

Reply:
Yeah, this note is unrelated. See my response on the #290 regarding that.